common/reviews/api/rush-lib.api.md

```diff
@@ -51,6 +56,21 @@ export class ApprovedPackagesPolicy {
     get reviewCategories(): Set<string>;
 }
 
+// @beta
+export class BuildCacheConfiguration {
+    readonly buildCacheEnabled: boolean;
+    cacheWriteEnabled: boolean;
+    readonly cloudCacheProvider: ICloudBuildCacheProvider | undefined;
+    // (undocumented)
+    static getBuildCacheConfigFilePath(rushConfiguration: RushConfiguration): string;
+    // Warning: (ae-forgotten-export) The symbol "GetCacheEntryIdFunction" needs to be exported by the entry point index.d.ts
+    readonly getCacheEntryId: GetCacheEntryIdFunction;
+    static loadAndRequireEnabledAsync(terminal: ITerminal, rushConfiguration: RushConfiguration, rushSession: RushSession): Promise<BuildCacheConfiguration>;
+    // Warning: (ae-forgotten-export) The symbol "FileSystemBuildCacheProvider" needs to be exported by the entry point index.d.ts
```

Comment:
Address

Reply:
Marked @internal to match the existing annotation on the class